diff --git a/node_modules/test-exclude/index.js b/node_modules/test-exclude/index.js
index aecb72d..88db210 100644
--- a/node_modules/test-exclude/index.js
+++ b/node_modules/test-exclude/index.js
@@ -4,6 +4,7 @@ const path = require('path');
 const { promisify } = require('util');
 const glob = promisify(require('glob'));
 const minimatch = require('minimatch');
+const matchesGlob = typeof minimatch === 'function' ? minimatch : minimatch.minimatch;
 const { defaults } = require('@istanbuljs/schema');
 const isOutsideDir = require('./is-outside-dir');
 
@@ -96,7 +97,7 @@ class TestExclude {
         }
 
         const dot = { dot: true };
-        const matches = pattern => minimatch(pathToCheck, pattern, dot);
+        const matches = pattern => matchesGlob(pathToCheck, pattern, dot);
         return (
             (!this.include || this.include.some(matches)) &&
             (!this.exclude.some(matches) || this.excludeNegated.some(matches))
